src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
```

Comment:
According to the checklist, classes must have access modifiers. Here, `Lottery` class doesn't have any. It's better to explicitly define it as public.